include/ccmath/internal/math/runtime/pp/simd.hpp

```diff
@@ -289,12 +303,12 @@ namespace ccm::pp
 		// ctors
 
 		template <class U>
-		constexpr basic_simd(U&& value) noexcept
+		constexpr basic_simd(U && value) noexcept
 		{
 		}
 
 		template <class U, class UAbi>
-		constexpr CCM_CPP20_EXPLICIT(true) basic_simd(basic_simd<U, UAbi> const& value) noexcept
+		constexpr CCM_CPP20_EXPLICIT(true) basic_simd(basic_simd<U, UAbi> const & value) noexcept
```

Comment:
Hmm, in my opinion it should be `const& value` rather than `const & value`. It might be your code style but it just looks like you're trying to perform bitwise and operation on const and "value" parameter.

Reply:
Ignore this section for now as it's still extremely experimental and has not been really defined yet. The plan though is the design is going to be somewhat inspired by std::simd in c++26.